CarpetX/src/driver.cxx

```diff
@@ -2167,7 +2168,14 @@ operator<<(YAML::Emitter &yaml,
   yaml << YAML::BeginMap;
   yaml << YAML::Key << "commongroupdata" << YAML::Value
        << (GHExt::CommonGroupData)arraygroupdata;
-  yaml << YAML::Key << "data" << YAML::Value << arraygroupdata.data;
+  yaml << YAML::Key << "data" << YAML::Value << YAML::Flow << YAML::BeginSeq;
+  for (int tl = 0; tl < int(arraygroupdata.data.size()); ++tl) {
+    yaml << YAML::BeginSeq;
+    for (int vi = 0; vi < arraygroupdata.numvars; ++vi)
+      yaml << *arraygroupdata.data.at(tl).dataPtr(vi);
```

Comment:
This seems to output a pointer?

Reply:
True, I'll update so that the data output is the same (kind of) data as is output for grid functions.